R/compute_variances.R

```diff
@@ -250,7 +250,7 @@
       vc = vcorr,
       re = list(lme4::ranef(x))
     )
-    names(vals$re) <- re_names[1:length(vals$re)]
+    names(vals$re) <- re_names[seq_along(vals$re)]
```

Comment:
Is this `seq_along(vals$re)` or `seq_along(length(vals$re))`?

Reply:
Without `length()` is the correct one: 

``` r
1:length(mtcars$qsec)
#>  [1]  1  2  3  4  5  6  7  8  9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25
#> [26] 26 27 28 29 30 31 32

seq_along(mtcars$qsec)
#>  [1]  1  2  3  4  5  6  7  8  9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25
#> [26] 26 27 28 29 30 31 32
```

<sup>Created on 2022-07-31 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

Reply:
Yes, I confused seq_along with seq_len.